content/02.introduction.md

```diff
@@ -75,13 +75,14 @@ Nevertheless, genomic data show some significant statistical evidence of ongoing
 There are two known variants of the spike protein that differ by a single amino acid at position 614 (G614 and D614), and there is evidence that G614 has already become dominant over D614 by June 2020 [@doi:10.1016/j.cell.2020.06.043].
 While there is a hypothesis that this genomic change increased the SARS-CoV-2 infectivity and virulence, there is no data yet to support or discard this hypothesis reliably [@doi:10.1016/j.cell.2020.06.040].
 The study [@doi:10.1016/j.meegid.2020.104351] has reported more than 198 recurrent mutations using a dataset of 7666 curated sequences showing the dynamic of SARS-CoV-2 evolution.
-Overall, while it is obvious that SARS-CoV-2 exhibits moderate potential for further evolution, much uncertainty remains about SARS-CoV-2's evolutionary trends and their consequences.
+Overall, while it is evident that SARS-CoV-2 exhibits moderate potential for further evolution, much uncertainty remains about SARS-CoV-2's evolutionary trends and their consequences due to a limited number of studies addressing SARS-CoV-2 ongoing evolution.
 
 The numerous mutations [@doi:10.1186/s12967-020-02344-6; @doi:10.1093/nsr/nwaa036; @doi:10.1016/j.meegid.2020.104351] observed in SARS-CoV-2 individual genome sequences can be used as signals during outbreak investigations and discovering of transmission patterns.
```

Comment:
```suggestion
The numerous mutations [@doi:10.1186/s12967-020-02344-6; @doi:10.1093/nsr/nwaa036; @doi:10.1016/j.meegid.2020.104351] observed in individual SARS-CoV-2 genome sequences signal transmission patterns and can be used during outbreak investigations.
```
Just a suggestion -- I think not everyone reading it will be familiar with phylogenetics and related problems, so I was trying to think of how to help lead them between your ideas -- maybe this would work?

Reply:
I agree. Thank you!